src/sentry/api/serializers/models/repository_project_path_config.py

```diff
@@ -9,13 +9,17 @@
 @register(RepositoryProjectPathConfig)
 class RepositoryProjectPathConfigSerializer(Serializer):
     def serialize(self, obj, attrs, user):
+        integration = obj.organization_integration.integration
+        provider = integration.get_provider()
         return {
             "id": six.text_type(obj.id),
             "projectId": six.text_type(obj.project_id),
             "projectSlug": obj.project.slug,
             "repoId": six.text_type(obj.repository.id),
             "repoName": obj.repository.name,
-            "organizationIntegrationId": six.text_type(obj.organization_integration_id),
+            "integrationId": six.text_type(integration.id),
+            "provider": integration.provider,
+            "providerName": provider.name,
```

Comment:
we could probably just return the `provider` since `provider.key` should be the same as `integration.provider`